README.md

```diff
@@ -22,14 +22,83 @@ Note: Edge debugging via [Edge DevTools Protocol](https://docs.microsoft.com/en-
 * Any features that aren't script debugging.
 
 ## Getting Started
-We are working on creating and publishing a VS Code Extension for this project.
+For use inside VS Code:
+1. [Install the extension](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge)
+2. Restart VS Code and open the folder containing the project you want to work on.
 
 For use inside Visual Studio:
 1. Install the latest [Windows Insider Preview](https://insider.windows.com/en-us/getting-started/) build.
```

Comment:
This is no longer applicable as "April 2018 Update" for Windows is now live.  Instead we should say, Install a supported version of Windows.